fl4health/utils/metrics.py

```diff
@@ -1,47 +1,158 @@
-from __future__ import annotations
```

Comment:
lol...where did this guy come from 😂

Reply:
I believe this required for classes that have methods that return instances of class and you are using annotations. I think this was relevant when defining the metric meter based on meter type but no longer required.

SO post: https://stackoverflow.com/questions/61544854/from-future-import-annotations

Reply:
Ah I see. I completely forgot we did that